### PR TITLE
Fedora 22 and rel-eng issue tracker updates

### PR DESCRIPTION
**Pull request by lsm5**
- fedora:latest points to fedora:22
- fedora rel-eng to be contacted in case of any troubles with
fedora:(22,21) and @fedora-cloud/docker-brew-fedora for fedora:(rawhide,20)

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

**Comment by lsm5** [comment]
ping @tianon @yosifkit

**Comment by tianon** [comment]
Can you please move the changes over to `content.md` instead? (https://github.com/docker-library/docs/blob/master/README.md#image-namereadmemd)

File: fedora/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The metalink `http://mirrors.fedoraproject.org` is used to automatically select
 
 # Supported Docker versions
 
-This image is officially supported on Docker version 1.6.2.
+This image is officially supported on Docker version 1.6.2 and later.
```
Comment (tianon) [inline fedora/README.md]: There are newer Docker versions than 1.6.2?  Where? :smile:

(RCs don't really count :+1:, but this part of this file is generated besides :wink:)

File: fedora/README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Documentation for this image is stored in the [`fedora/` directory](https://gith
 
 ## Issues
 
-If you have any problems with or questions about this image, please contact us by filing a bug on [Fedora's bugzilla page](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora) (choose `docker-io` as component and include details about image problems in the description) or through a [GitHub issue](https://github.com/lsm5/docker-brew-fedora/issues).
+If you have any problems with or questions about this image, please file a ticket with [Fedora Release Engineering](https://fedorahosted.org/rel-eng/newticket) for `fedora:22` and `fedora:21` or file a [GitHub issue](https://github.com/fedora-cloud/docker-brew-fedora/issues/new) for `fedora:rawhide` and `fedora:20`.
```
Comment (tianon) [inline fedora/README.md]: This URL needs to change in https://github.com/docker-library/docs/blob/f68b2f970b50ae0af376a43d3bd03de59cd99d4e/update.sh#L34, and the rest needs to change in either the existing `mailing-list.md`, or in a new `user-feedback.md`.

File: fedora/README.md (hunk 1 above)
Comment (lsm5) [inline fedora/README.md]: umm, docker on rawhide is already 1.7.0, and by officially I guess I meant fedora, but maybe you mean docker :|

File: fedora/README.md (hunk 2 above)
Comment (lsm5) [inline fedora/README.md]: alright, will update this

File: fedora/README.md (hunk 1 above)
Comment (tianon) [inline fedora/README.md]: What's this Fedora that you speak of?  Is that a Docker image I can
download somewhere? :innocent:

Comment (lsm5) [inline fedora/README.md]: rawhide builds from master ...so ya ..anyway ... I'll shut up now :)

Comment (tianon) [inline fedora/README.md]: I'm just giving you a hard time. :heart:

I'm not opposed at all to adding this blurb in the template itself -- it
should be in "update.sh" or ".template-helpers" somewhere (can't remember
off-hand which has it).

**Comment by tianon** [comment]
ping @lsm5

**Comment by lsm5** [comment]
ugh, 2-month anniversary :| ...

@maxamillion ping, would you like to take this one? I can probably move this ahead if rel-eng procedures haven't changed since, but still be great to have you review it. lmk ...

**Comment by tianon** [comment]
Does this still need to be carried forward? @lsm5 @maxamillion

**Comment by lsm5** [comment]
i've been out of this loop for a while :| , i'll let @maxamillion comment

**Comment by tianon** [comment]
Ok, fair enough; thanks! :+1:

**Comment by maxamillion** [comment]
I somehow missed this a very long time ago. I'd like to close this in favor of https://github.com/docker-library/docs/pull/481 if that's alright with @lsm5

**Comment by lsm5** [comment]
@maxamillion yup, sure thing, thanks!

**Pull request closed without merging**